lib/pure/json.nim

```diff
@@ -817,7 +817,11 @@ proc parseJson(p: var JsonParser): JsonNode =
     p.a = ""
     discard getTok(p)
   of tkInt:
-    result = newJInt(parseBiggestInt(p.a))
+    try:
+      result = newJInt(parseBiggestInt(p.a))
+    except:
```

Comment:
Shouldn't this name the explicit exception?

Reply:
Yeah, please add an exception here.

Reply:
Done